pkg/webhook/admission/pod/mutator.go

```diff
@@ -33,6 +32,13 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/webhook/admission/types"
 )
 
+const (
+	StorageInitializerDefaultMemoryRequest string = "200Mi"
+	StorageInitializerDefaultMemoryLimit   string = "1Gi"
+	StorageInitializerDefaultCPURequest    string = "100m"
+	StorageInitializerDefaultCPULimit      string = "1"
+)
```

Comment:
do we need two copies of values both in constant and configmap ? We can just leave the defaults in config map.

Reply:
That's a good point :) I do have a question though. If a user has an overlay for the StorageInitializer configuration but forgot to configure CPU/Memory Limit/Request. Will kustomize use the values from the default configmap?

Reply:
I only kept the parameters from the configmap now :)